app/controllers/application_controller.rb

```diff
@@ -3,6 +3,7 @@ class ApplicationController < ActionController::Base
   before_action :configure_permitted_parameters, if: :devise_controller?
 
   def configure_permitted_parameters
-    devise_parameter_sanitizer.permit(:sign_up, keys: [:first_name, :last_name, :phone_number, :email, :country_id, :password])
+    devise_parameter_sanitizer.permit(:sign_up, keys: [:photo, :first_name, :last_name, :phone_number, :email, :country_id, :password])
+    devise_parameter_sanitizer.permit(:account_update, keys: [:photo, :email, :password])
   end
```

Comment:
Do these params sanitizers all run, or just one of them?
I don't understand it completly, but if you tested it, I trust you...

Reply:
First line is for signing up (new and create) and the second line is for editing your account (patch). That's how I understand them. @raffoz @Flott2410 ??